src/main/core/Resources/modules/user/store/actions.js

```diff
@@ -6,7 +6,7 @@ export const actions = {}
 
 actions.updatePassword = (user, plainPassword) => ({
   [API_REQUEST]: {
-    url: ['apiv2_user_update', {id: user.id}],
+    url: ['apiv2_user_update', {id: user.id, event: 'updatePassword'}],
```

Comment:
This and relevant code in the api should be removed.
To manage this, you should use `UserCrud::preCreate()` or `UserCrud::postCreate()` and compare the `plainPassword` values to know if you need to dispatch the event.

https://github.com/claroline/Claroline/blob/13.x/src/main/core/API/Crud/User/UserCrud.php

Reply:
Done, thx :)